acoustid/data/source.py

```
@@ -22,32 +23,63 @@ def get_source(conn: AppDB, id: int) -> dict[str, Any] | None:
-    Find a source in the database, create it if it doesn't exist yet.
-    """
-    query = sql.select(schema.source.c.id).where(
+) -> int | None:
+    query = sql.select(
+        schema.source.c.id,
+    ).where(
         sql.and_(
             schema.source.c.account_id == account_id,
             schema.source.c.application_id == application_id,
             schema.source.c.version == version,
         )
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)